Client/Application/Delegates/AppDelegate.swift

```diff
@@ -64,6 +66,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.window = UIWindow(frame: UIScreen.main.bounds)
         self.window!.backgroundColor = .black
         
+        //Brave Core Initialization
+        self.braveCore = BraveCoreMain()
+        self.braveCore?.setUserAgent(UserAgent.mobile)
```

Comment:
FYI the latest iOS safari user agent contains `14_0_1` not `14_0` https://github.com/brave/brave-ios/blob/ba8686d7436cf305693cdce32a3bfe58147a7fa0/Shared/UserAgentBuilder.swift#L99

Reply:
This is done manually now, added ticket for it, i will get it done in 1.22 or later, depends on time
https://github.com/brave/brave-ios/issues/3020